src/Engine/Audio/SteamAudio.cpp

```diff
@@ -1,59 +1,87 @@
 #include "SteamAudio.hpp"
+#include <assert.h>
 
 #ifdef USINGMEMTRACK
 #include <MemTrackInclude.hpp>
 #endif
 
+using namespace Audio;
+
 SteamAudio::SteamAudio() {
-    context = nullptr;
-    environmentalRenderer = nullptr;
-}
+    playerPos = IPLVector3{ 0,0,0 };
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)